dkg/disk.go

```diff
@@ -54,6 +54,8 @@ func loadDefinition(ctx context.Context, conf Config) (cluster.Definition, error
 		if err != nil {
 			return cluster.Definition{}, errors.Wrap(err, "read definition")
 		}
+
+		log.Info(ctx, "Definition file loaded from remote URI", z.Str("URI", conf.DefFile))
```

Comment:
move logging of definition hash here (remove from later log)

Reply:
suggest: `Cluster definition downloaded from URL`

URL is more accurate since we only allow HTTP and more common

Reply:
done